performance_regression_cdc_test.py

```diff
@@ -0,0 +1,269 @@
+                               test_name="test_write_profile",
+                               sub_type="cdc_disabled")
+
+        node1: BaseNode = self.db_cluster.nodes[0]
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1)
+
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_write_profile",
+                               sub_type="cdc_enabled")
+
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1, cdclog_table=True)
+
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'preimage': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_write",
+                               sub_type="cdc_preimage_enabled")
+
+        self.wait_no_compactions_running()
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def test_write(self):  # pylint: disable=unused-variable
+        self.keyspace = "keyspace1"
+        self.table = "standard1"
+        write_cmd = self.params.get("stress_cmd_w")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_write",
+                               sub_type="cdc_disabled")
+
+        node1: BaseNode = self.db_cluster.nodes[0]
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1)
+
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_write",
+                               sub_type="cdc_enabled")
+
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+        self.truncate_base_table(node1, cdclog_table=True)
+
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'preimage': true}}")
+
+        _ = self._workload_cdc(write_cmd, 2,  # pylint: disable=unused-variable
+                               test_name="test_write",
+                               sub_type="cdc_preimage_enabled")
+
+        self.wait_no_compactions_running()
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def test_mixed(self):  # pylint: disable=too-many-locals,unused-variable
+        self.keyspace = "keyspace1"
+        self.table = "standard1"
+        write_cmd = self.params.get("stress_cmd_w")
+        read_cdc_log_cmd = self.params.get("stress_cmd_r")
```

Comment:
`stress_cmd_r` isn't in any of the yamls. again, can be added in it's own PR when used.

Reply:
moved to another PR